index.html

```diff
@@ -57,5 +57,202 @@ <h1>I’m Happi. Glad to see you!</h1>
+            <img src="./figmaImages/icons/circle-icon.svg" alt=""> 
+            <span class="subheading-list-title">Back End Dev</span>
+            <img src="./figmaImages/icons/circle-icon.svg" alt="">
+            <span class="subheading-list-title">2015</span>
+            </section>
+            <section class="card-content">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or 
+                    sign-ups required.
+                </p>
+            </section>
+            <ul class="webdev-languages">
+                <li> <a href="#html">HTML</a> </li>
+                <li> <a href="#css">CSS</a> </li>
+                <li> <a href="#javascript">Javascript</a></li>
+            </ul>
+            <button class="see-project">See Project</button>
+        </section>
+        </section>
+    <section class="cards">
+        <img src="./figmaImages/snapshoots/Snapshoot Portfolio-3.svg" alt="Snapshoot">
+   
+    <div class="cards-description">
+        <h2 class="cards-titles">Tonic</h2>
+        <div class="subheading-lists">
+        <span class="canopy">CANOPY</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">Back End Dev</span>
+        <img src="./figmaImages/icons/circle-icon.svg" alt="">
+        <span class="subheading-list-title">2015</span>
+        </div>
+        <section class="card-content">
+            <p>
+                A daily selection of privately personalized reads; no accounts or 
+                sign-ups required.
+            </p>
+        </section>
+        <ul class="webdev-languages">
+            <li> <a href="#html">HTML</a> </li>
+            <li> <a href="#css">CSS</a> </li>
+            <li> <a href="#javascript">Javascript</a></li>
+        </ul>
+        <button class="see-project">See Project</button>
+    </div>
+    </section>
+<section class="cards">
+    <img src="./figmaImages/snapshoots/Snapshoot Portfolio-4.svg" alt="Snapshoot">
+
+<div class="cards-description">
+    <h2 class="cards-titles">Multi-Post
+        Stories</h2>
+    <div class="subheading-lists">
+    <span class="canopy">CANOPY</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">Back End Dev</span>
+    <img src="./figmaImages/icons/circle-icon.svg" alt="">
+    <span class="subheading-list-title">2015</span>
+    </div>
+    <section class="card-content">
+        <p>
+            A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.
+        </p>
+    </section>
+    <ul class="webdev-languages">
+        <li> <a href="#html">HTML</a> </li>
+        <li> <a href="#css">CSS</a> </li>
+        <li> <a href="#javascript">Javascript</a></li>
+    </ul>
+    <button class="see-project">See Project</button>
+</div>
+</section>
+</section>
+<section class="about-myself">
+    <h1>About Myself</h1>
```

Comment:
- [ ] There should only be one `h1` tag for proper page structure which we have already added to the introductory section. All subsequent headings in sections to follow should be `hx` tags e,g, h2, h3, h4. Kindly change this tag to `h2`

Reply:
Okey
Thanks